clients/src/main/java/org/apache/kafka/common/requests/AssignReplicasToDirsRequest.java

```diff
@@ -27,6 +27,8 @@
 
 public class AssignReplicasToDirsRequest extends AbstractRequest {
 
+    public static final int MAX_ASSIGNMENTS_PER_REQUEST = 2250;
```

Comment:
Can you add JavaDoc about this? Including the part where we want to keep it under 64kb.